front_end/src/app/(main)/questions/helpers/filters.ts

```diff
@@ -247,13 +251,29 @@ export function getFilterSectionParticipation({
     options: [
       {
         id: POST_FORECASTER_ID_FILTER,
-        label: t("predicted"),
+        label: t("searchOptionPredicted"),
         value: user.id.toString(),
         active: !!params.get(POST_FORECASTER_ID_FILTER),
       },
+      {
+        id: POST_WITHDRAWN_FILTER,
+        label: t("searchOptionActivePrediction"),
+        value: false.toString(),
+        active:
+          !!params.get(POST_FORECASTER_ID_FILTER) &&
+          params.get(POST_WITHDRAWN_FILTER) === "false",
+      },
+      {
+        id: POST_WITHDRAWN_FILTER,
+        label: t("searchOptionWithdrawnPrediction"),
+        value: true.toString(),
+        active:
+          !!params.get(POST_FORECASTER_ID_FILTER) &&
+          params.get(POST_WITHDRAWN_FILTER) === "true",
+      },
```

Comment:
@ncarazon I could use your help here.
I wanted to add 2 additional options - one for when a user has prediction and their prediction is still active, and another when they withdrew their most recent forecast.
The URL params are set up on the backend to support things like: `?forecaster_id=12345&withdrawn=true`

But I can't quite figure out how to do that with the filter components we have available. I think it'll take you 1/10th the time it would take me. Could you help me out?

Reply:
Pushed a commit. The solution is a bit hacky, but in order to properly support multiple filters being applied per a single option in the popup we would need to do a refactor of the component and adjust it in a bunch of places, which will take some time. So I've left a TODO to consider refactoring when there is time